src/colorize.cr

```diff
@@ -1,3 +1,88 @@
+# The Colorize library allows you to render texts with the given color and text decoration.
```

Comment:
"With Colorize you can change the fore- and background colors and text decorations when rendering text on terminals supporting ANSI escape codes. It adds the `colorize` method to `Object` and thus all classes as its main interface. Its first argument changes the foreground color:"